robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -629,7 +972,19 @@ public class ShadowAccessibilityNodeInfo {
     }
     areEqual = areEqual && (parent == otherShadow.parent);
 
-    areEqual = areEqual && (actionFlags == otherShadow.actionFlags);
+    areEqual = areEqual && (propertyFlags == otherShadow.propertyFlags);
+
+#if ($api >= 21)
+    boolean actionsArrayEquality = false;
+    if (actionsArray == null && otherShadow.actionsArray == null) {
+      actionsArrayEquality = true;
+    } else {
+      actionsArrayEquality = actionsArray.equals(otherShadow.actionsArray);
+    }
+    areEqual = areEqual && actionsArrayEquality;
```

Comment:
Can we replace:

    areEqual = areEqual && 

with

    areEqual &=

throughout this method?